index.html

```diff
@@ -3,18 +3,46 @@
     <head>
         <meta charset="utf-8">
         <meta name="google-site-verification" content="XYxEjxEMJ-lflwv40c2SiK5UsgTeMnMN0xrWCLww1es">
-        <link href='//fonts.googleapis.com/css?family=Open+Sans:300' rel='stylesheet' type='text/css'>
         <link href="css/index.css" rel="stylesheet" type="text/css">
         <title>Copilot</title>
     </head>
     <body>
+        <div id="info">
+            <div id="info-wrapper">
+                <center>
+                    <h1>About Copilot</h1>
+                    <br />
+                </center>
+                    <p>
+                        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bacon ipsum dolor amet kielbasa chuck short loin, tenderloin sirloin beef ribs biltong jerky ground round leberkas pork chop. Hamburger ball tip turducken, chuck cupim bresaola beef pork loin prosciutto jerky pig boudin. Filet mignon pork chop frankfurter, pork loin kevin tri-tip porchetta spare ribs turducken andouille. Corned beef ground round andouille, pancetta boudin meatloaf ribeye meatball landjaeger cow filet mignon strip steak porchetta drumstick kevin. Swine ham landjaeger, venison fatback andouille bacon alcatra tail doner pork belly rump beef hamburger. Andouille shank landjaeger doner porchetta rump pancetta.
```

Comment:
Woah that's a lot of spaces. If you want an indent like that maybe add it to the CSS using ```:before```?